tests/unit/services/cookies-test.js

```diff
@@ -235,18 +237,29 @@ describeModule('service:cookies', 'CookiesService', {}, function() {
 
   describe('in the FastBoot server', function() {
     beforeEach(function() {
+
+      let request = EmberOject.extend({
+        init() {
+          this._super(...arguments);
+          this.cookies = {};
+          this.protocol;
+          this._host;
```

Comment:
I guess the 2 previous lines aren't actually needed (unless this is a JS feature I'm unaware of ;)?